src/controllers/accounts/helpers.js

```diff
@@ -149,7 +149,13 @@ helpers.getUserDataByUserSlug = function (userslug, callerUID, callback) {
 			userData.birthday = validator.escape(String(userData.birthday || ''));
 			userData.moderationNote = validator.escape(String(userData.moderationNote || ''));
 
-			userData['cover:url'] = userData['cover:url'] ? (userData['cover:url'].startsWith('http') ? userData['cover:url'] : (nconf.get('relative_path') + userData['cover:url']) ) : require('../../coverPhoto').getDefaultProfileCover(userData.uid);
+			var _userDataCoverUrl;
+			if(userData['cover:url']) {
```

Comment:
This needs to be `if (userData['cover:url']) {`, 😆

Reply:
It would be easier if you just run `npm run lint` locally to get all the linting errors.